agent/state.mjs

```diff
@@ -0,0 +1,306 @@
+
+        // true - just show configuration values and exit
+        "bShowConfigMode": false,
+
+        "isEnabledMultiCall": true,
+
+        "bNoWaitSChainStarted": false,
+        "nMaxWaitSChainAttempts": 0 + Number.MAX_SAFE_INTEGER, // 20
+
+        "nAmountOfWei": 0,
+        "nAmountOfToken": 0,
+        "arrAmountsOfTokens": null,
+        "idToken": 0,
+
+        "nTransferBlockSizeM2S": 4,
+        "nTransferBlockSizeS2M": 4,
+        "nTransferBlockSizeS2S": 4,
+        "nTransferStepsM2S": 8,
+        "nTransferStepsS2M": 8,
+        "nTransferStepsS2S": 8,
+        "nMaxTransactionsM2S": 0,
+        "nMaxTransactionsS2M": 0,
+        "nMaxTransactionsS2S": 0,
+
+        "nBlockAwaitDepthM2S": 0,
+        "nBlockAwaitDepthS2M": 0,
+        "nBlockAwaitDepthS2S": 0,
+        "nBlockAgeM2S": 0,
+        "nBlockAgeS2M": 0,
+        "nBlockAgeS2S": 0,
+
+        "nLoopPeriodSeconds": 10,
+
+        "nNodeNumber": 0, // S-Chain node number(zero based)
+        "nNodesCount": 1,
+        "nTimeFrameSeconds": 0, // 0-disable, 60-recommended
+        "nNextFrameGap": 10,
+
+        "nAutoExitAfterSeconds": 3600, // 0-disable
+
+        "jo_community_pool": null, // only main net
+        "jo_deposit_box_eth": null, // only main net
+        "jo_deposit_box_erc20": null, // only main net
+        "jo_deposit_box_erc721": null, // only main net
+        "jo_deposit_box_erc1155": null, // only main net
+        "jo_deposit_box_erc721_with_metadata": null, // only main net
+        "jo_linker": null, // only main net
+
+        "isWithMetadata721": false,
+
+        "jo_token_manager_eth": null, // only s-chain
+        // "jo_token_manager_eth_target": null, // only s-chain target
+        "jo_token_manager_erc20": null, // only s-chain
+        "jo_token_manager_erc20_target": null, // only s-chain
+        "jo_token_manager_erc721": null, // only s-chain target
+        "jo_token_manager_erc721_target": null, // only s-chain target
+        "jo_token_manager_erc1155": null, // only s-chain
+        "jo_token_manager_erc1155_target": null, // only s-chain target
+        "jo_token_manager_erc721_with_metadata": null, // only s-chain target
+        "jo_token_manager_erc721_with_metadata_target": null, // only s-chain target
+        "jo_community_locker": null, // only s-chain
+        "jo_community_locker_target": null, // only s-chain target
+        "jo_message_proxy_main_net": null,
+        "jo_message_proxy_s_chain": null,
+        "jo_message_proxy_s_chain_target": null, // only s-chain target
+        "jo_token_manager_linker": null,
+        "jo_token_manager_linker_target": null, // only s-chain target
+        "eth_erc20": null, // only s-chain
+        // "eth_erc721": null, // only s-chain
+        // "eth_erc1155": null, // only s-chain
+        "eth_erc20_target": null, // only s-chain target
+        // "eth_erc721_target": null, // only s-chain target
+        // "eth_erc1155_target": null, // only s-chain target
```

Comment:
Switch to camelCase, please and remove redundant comments

Reply:
Fixed